src/VonageServiceProvider.php

```diff
@@ -187,18 +180,11 @@ protected function vonageConfigHasNo($key): bool
      */
     protected function vonageConfigHas($key): bool
     {
-        /** @var Config $config */
-        $config = $this->app->make(Config::class);
-
-        // Check for Nexmo config file.
-        if ( ! $config->has('vonage')) {
+        if (!array_key_exists($key, $this->config)) {
```

Comment:
Is this necessary, given line 187 exists?

Reply:
it's a PHP quirk that instead of returning null or false - if you pass in an array key that doesn't exist it'll throw a fatal. So we need to check both the array key is there AND it has a value